src/dht_proxy_server.cpp

```diff
@@ -419,18 +419,19 @@ DhtProxyServer::handlePushListeners()
             auto token = pushListener->token;
             auto callbackId = pushListener->callbackId;
             auto isAndroid = pushListener->isAndroid;
-            auto internalToken = std::move(dht_->listen(pushListener->hash,
+            auto internalToken = dht_->listen(pushListener->hash,
                 [this, key, callbackId, token, isAndroid](std::shared_ptr<Value> /*value*/) {
                     // Build message content.
                     Json::Value json;
                     if (callbackId > 0) {
                         json["callback_id"] = callbackId;
                     }
+                    json["to"] = callbackId;
```

Comment:
@aberaud  pushClientId here? callbackId is already given.

Reply:
done